ansible_collections/arista/cvp/plugins/module_utils/device_tools.py

```diff
@@ -1587,13 +1587,13 @@ def apply_bundle(self, user_inventory: DeviceInventory):
                                 "Error applying bundle to device: %s", str(catch_error)
                             )
                             self.__ansible.fail_json(
-                                msg="Error applying bundle to device"
+                                msg="Error applying bundle to device "
                                 + device.fqdn
                                 + ": "
-                                + catch_error
+                                + str(catch_error)
                             )
                         else:
-                            if resp["data"]["status"] == "success":
+                            if resp and resp["data"]["status"] == "success":
```

Comment:
so this code modification was added because of some error that was triggered?

Reply:
No, found it in the code only that this case is also possible

Reply:
> so this code modification was added because of some error that was triggered?

This was added as a safeguard. We are not calling here with `create_task=False`, we will not run into resp being None here. I am open to your suggestion on whether we need to check if resp is None or not.